addon/components/validated-input/types/-themes/bootstrap/checkbox-group.js

```diff
@@ -0,0 +1,6 @@
+import layout from "../../../../../templates/components/validated-input/types/-themes/bootstrap/checkbox-group";
```

Comment:
Is this worth using relative paths here? Maybe a bit nicer to read with `ember-validated-form/templates/components/validated-input/types/-themes/bootstrap/checkbox-group`.

Reply:
Or is the relative path required since its a template?

Reply:
I'm not sure, I just followed the same pattern used in the other files. If we decide to refactor this, I'd suggest to do it in a separate PR. cc @anehx

Reply:
I discussed this with @anehx earlier, he said that refactoring the components to octane should solve this because of component/template colocation.